src/amo/reducers/versions.js

```diff
@@ -101,12 +126,22 @@ type GetBySlugParams = {|
 export const getVersionsBySlug = ({
   slug,
   state,
-}: GetBySlugParams): Array<AddonVersionType> | null => {
+}: GetBySlugParams): Array<AddonVersionType> => {
   invariant(slug, 'slug is required');
   invariant(state, 'state is required');
 
   const infoForSlug = state.bySlug[slug];
-  return (infoForSlug && infoForSlug.versions) || null;
```

Comment:
Why does this no longer return `null` for non-existent versions? It seems like that leaves no way to detect if versions by slug have been loaded or not. I haven't seen the component patches for this so maybe it's fine?

Reply:
You're right - a consumer may want to be able to differentiate between null and no records found. I've changed it back.